packages/manager/apps/public-cloud/src/index.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html class="h-100">
+<html class="theme-default h-100">
```

Comment:
Maybe we should prefix or rename this one, like ovh-theme-light to avoid conflict with others librairies ?